src/main/cre.js

```diff
@@ -184,14 +184,24 @@ function showPop(type) {
   function getHeaderRow() {
     var headerHTML;
     if (locationName === "Event") {
-      headerHTML =
-        "<tr align='left'><th align='left'>Mouse Name</th><th data-filter='false'>Catch Rate</th><th data-filter='false'>Gold</th><th data-filter='false'>Points</th><th data-filter='false'>Min Luck</th>";
```

Comment:
This part is not related to any function change, just OCD'ing on these 2 >120 character lines. If you prefer I can remove them here and issue a separate PR for them.